docs/getting-started/installation/helm.md

```diff
@@ -117,6 +117,8 @@ You have to manually delete custom resource definitions created by the `helm ins
     kubectl delete crd sbomreports.aquasecurity.github.io
     kubectl delete crd clustersbomreports.aquasecurity.github.io
     kubectl delete crd clustervulnerabilityreports.aquasecurity.github.io
+    kubectl delete crd clustersbomreports.aquasecurity.github.io
+    kubectl delete crd clustervulnerabilityreports.aquasecurity.github.io
```

Comment:
It's already above, right?

Reply:
True. To be honest, I don't remember why I thought this patch was necessary.